packages/core/src/card/Card.css

```diff
@@ -8,6 +8,11 @@
   padding: var(--saltCard-padding, var(--salt-spacing-300));
 }
 
+.saltCard-secondary {
```

Comment:
Should `primary` variant sharing same API?

Reply:
yeah good point, I will update this

Reply:
`--saltCard-background` not the same after update?

Reply:
I've updated - is this what you mean? or should it be a different token like:

```
background: var(--saltCard-secondary-background, var(--salt-container-secondary-background));
```